src/main/java/com/puppycrawl/tools/checkstyle/checks/header/RegexpHeaderCheck.java

```diff
@@ -115,6 +117,20 @@ protected void processFiltered(File file, FileText fileText) {
         }
     }
 
+    /**
+     * Returns the line from the header. Where the line is blank return the regexp pattern
+     * for a blank line.
+     * @param headerLineNo heaeder line number to return
+     * @return the line from the header
+     */
+    private String header(int headerLineNo) {
```

Comment:
Method name should start with a verb.
`header` => `getHeaderLine`

Reply:
@rnveach Done.